packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx

```diff
@@ -106,79 +88,60 @@ const DialogueSubmit = ({
   const submitWithoutConfirmation = () => collectionName === "Posts" && updateCurrentValues({draft: false});
 
   const requireConfirmation = isLW && collectionName === 'Posts' && !!document.debate;
+  const showShortformButton = !!userShortformId && !isFriendlyUI;
 
   const onSubmitClick = requireConfirmation ? submitWithConfirmation : submitWithoutConfirmation;
 
-
+  const {Row} = Components;
   return (
-    <React.Fragment>
```

Comment:
This component was very messy with a ton of unused stuff, plus it tried to add class-component-esque `PropTypes` even though this is a functional component which confused the typechecker so I removed those too. The only material change in this file is removing the "post to shortform" button.